JSON.sh

```diff
@@ -83,11 +83,11 @@ tokenize () {
   if echo "test string" | egrep -o "test" >/dev/null 2>&1
   then
     ESCAPE='(\\[^u[:cntrl:]]|\\u[0-9a-fA-F]{4})'
-    CHAR='[^[:cntrl:]"\\]'
+    CHAR='([^[:cntrl:]"\\]|)'
```

Comment:
Pay ~~no~~ attention to the UTF-8 behind the curtain...

![image](https://cloud.githubusercontent.com/assets/1199584/19753650/76323286-9bd4-11e6-8ad6-93e90e5fdfc3.png)